src/proto/connection.rs

```diff
@@ -106,10 +106,6 @@ where
     pub fn new(codec: Codec<T, Prioritized<B>>, config: Config) -> Connection<T, P, B> {
         fn streams_config(config: &Config) -> streams::Config {
             streams::Config {
-                local_init_window_sz: config
-                    .settings
-                    .initial_window_size()
-                    .unwrap_or(DEFAULT_INITIAL_WINDOW_SIZE),
```

Comment:
Oh good catch. This gets applied after the peer sends back an ACK, right?

Reply:
Yes, it will: 

https://github.com/hyperium/h2/blob/dd67adc19268a9780b09502ecc660be8363f008f/src/proto/streams/recv.rs#L489

We're still negotiating, but just not eagerly opening streams with non-default settings.